contents/blog/github-cms.md

```diff
@@ -0,0 +1,75 @@
+
+2. Issues go into a content [project board](https://docs.github.com/en/issues/tracking-your-work-with-issues/planning-and-tracking-work-for-your-team-or-project#adding-issues-to-a-project-board) with columns for the backlog, coming soon (in priority order), in progress, in review, and done.
+
+3. Content gets worked on. When done, the creator opens a pull request (with a [checklist](https://github.com/PostHog/posthog.com/blob/master/.github/pull_request_template.md)), and asks for reviews. [GitHub Actions](/blog/automating-a-software-company-with-github-actions) run for spellcheck and Vercel site previews.
+
+4. Reviewers comment and make suggestions. Creator updates until approved.
+
+5. Content gets merged and changes publish to the production site automatically. PRs are connected to the issues, which are automatically closed when merged.
+
+GitHub and a text editor are all we need to go from idea to published content. The features of GitHub provide many benefits to our workflow such as:
+
+- Automation for templates, checklists, spellcheck, review notifications, and site previews.
+
+- Central location for ideas, drafts, team comments, finished content, and revisions.
+
+- Tools for feedback including suggestions, full markdown comments, and comment history.
+
+The one thing it lacks is a content calendar, but it's not something we miss much. Most content can be published as soon as it's ready. If the content relates to a launch, we create a branch with all the changes (including documentation and landing page updates) and merge it as soon as the feature launches.
+
+## 3. GitHub keeps us engineering-focused
+
+At PostHog, [everybody codes](/handbook/company/values#everyone-codes), and our [ideal customers](/blog/creating-ideal-customer-profile) are developers. Using GitHub and combining content and code keeps this top of mind. GitHub is the ideal CMS for engineers and developers because they already use it, and it encourages non-technical people to be more technical.
+
+Developers don’t want to spend time in a CMS. They want to use tools they already know and spend time in. Trying to get a busy developer to contribute on GitHub is easier than trying to get them to contribute in a CMS they aren’t familiar with. This means they make more changes to our site and review content more often. **~40% of [merged pull requests in the posthog.com repo](https://github.com/PostHog/posthog.com/pulls?q=is%3Aclosed) are from people outside of marketing, website, and docs.**
+
+It also encourages our marketing team to be more technical. They must understand how the code and content interact. For example, missing metadata causes an error in the web app. The marketing team can figure out and solve this themselves, rather than asking the development team to do it.
```

Comment:
```suggestion
It also encourages our marketing team to be more technical. They must understand how the code and content interact. For example, when missing metadata causes an error in the web app. The marketing team can figure out and solve this themselves, rather than asking the development team to do it.
```